src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -176,6 +176,33 @@ public static KeyToValueMappingEstimator MapKeyToValue(this TransformsCatalog.Co
             params (string outputColumnName, string inputColumnName)[] columns)
             => new ValueMappingEstimator<TInputType, TOutputType>(CatalogUtils.GetEnvironment(catalog), keys, values, columns);
 
+        /// <summary>
+        /// <see cref="ValueMappingEstimator"/>
+        /// </summary>
+        /// <typeparam name="TInputType">The key type.</typeparam>
+        /// <typeparam name="TOutputType">The value type.</typeparam>
+        /// <param name="catalog">The categorical transform's catalog</param>
```

Comment:
Can you use the comment from the estimator constructor, I believe that this is the type of the items here.